src/components/Modal/Modal.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import styled from 'styled-components'
+import EscapeOutside from '../EscapeOutside/EscapeOutside'
+import { Button, Text, breakpoint } from '@aragon/ui'
+
+const Modal = ({ title, body, moreText, onHide, More, blocking }) => (
+  <WrapModal role="alertdialog" onEscapeOutside={!blocking && onHide}>
```

Comment:
⚠️ `onEscapeOutside={!blocking && onHide}` can be two types (`boolean` or `func`); I'd propose just making it a `func` by using a ternary: `blocking ? noop : onHide`